### DISCUSSION TITLE
Modularity

### DISCUSSION THREAD

**Discussion by reddeerishere** [mdbootstrap/TW-Elements#1515] [Share ideas & request features]
Are you planning to add JS files optimization with ES Modules to Elements, like you did in MDBootstrap?

I think it's a must-have.

**Comment by filipkappa** [comment]
We've added the modularity feature in our latest release #1586 !

You can learn more about optimization of your Tailwind project here:
https://tailwind-elements.com/docs/standard/getting-started/optimization/
